src/bindings/html/view.js

```diff
@@ -104,23 +104,30 @@ export function translateDocument(view, langs) {
     langs[0].code === html.getAttribute('lang') ?
       Promise.resolve() :
       translateFragment(view, langs, html).then(
-        () => setDOMAttrs(html, langs));
+        () => setLangDir(html, langs));
 
-  return translated.then(
-    () => readiness.set(html, true));
+  return translated.then(() => {
+    setLangs(html, langs);
+    readiness.set(html, true);
+  });
 }
 
-function setDOMAttrsAndEmit(html, langs) {
-  setDOMAttrs(html, langs);
+function setLangs(html, langs) {
+  const codes = langs.map(lang => lang.code);
+  html.setAttribute('langs', codes.join(' '));
```

Comment:
I'm still not sure if it should be space or comma separated list. Our meta use comma and we did this for a reason.